modules/021-cni-cilium/images/builder/werf.inc.yaml

```diff
@@ -0,0 +1,157 @@
+artifact: {{ $.ModuleName }}/{{ $.ImageName }}-envoy-artifact
+from: quay.io/cilium/cilium-envoy:04413917ff99e4f6ab51d1c6eb424d4a055f4462@sha256:af076f80818bc8d894f2f4f3104d5f4288112a67be5fb6e1b9a9c78370c7c9c8
+---
+artifact: {{ $.ModuleName }}/{{ $.ImageName }}-cert-artifact
+from: {{ $.Images.BASE_ALPINE }}
+shell:
+  beforeInstall:
+    - apk add --no-cache ca-certificates
+---
+image: {{ $.ModuleName }}/{{ $.ImageName }}-artifact
+from: {{ $.Images.BASE_GOLANG_19_BULLSEYE }}
+import:
+- artifact: {{ $.ModuleName }}/{{ $.ImageName }}-llvm-artifact
+  add: /usr/local/bin/llvm-objcopy
+  to: /bin
+  before: install
+mount:
+- fromPath: ~/go-pkg-cache
+  to: /go/pkg
+shell:
+  beforeInstall:
+  - apt-get update
+  - apt-get upgrade -y --no-install-recommends
+  - |
+    apt-get install -y --no-install-recommends \
+    gcc-aarch64-linux-gnu \
+    g++-aarch64-linux-gnu \
+    libc6-dev-arm64-cross \
+    binutils-aarch64-linux-gnu \
+    gcc-x86-64-linux-gnu \
+    g++-x86-64-linux-gnu \
+    libc6-dev-amd64-cross \
+    binutils-x86-64-linux-gnu \
+    unzip \
+    binutils \
+    coreutils \
+    curl \
+    gcc \
+    git \
+    libc6-dev \
+    patch \
+    make
+  - apt-get purge --auto-remove
+  - apt-get clean
+  - rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
+  - mkdir -p chmod +x /go/src/github.com/cilium/cilium/images/builder
+  - curl -sSfL https://raw.githubusercontent.com/cilium/cilium/v{{ $ciliumVersion }}/images/builder/install-protoc.sh -o /go/src/github.com/cilium/cilium/images/builder/install-protoc.sh
+  - curl -sSfL https://raw.githubusercontent.com/cilium/cilium/v{{ $ciliumVersion }}/images/builder/install-protoplugins.sh -o /go/src/github.com/cilium/cilium/images/builder/install-protoplugins.sh
+  - chmod +x /go/src/github.com/cilium/cilium/images/builder/*.sh
+  install:
+  - cd /go/src/github.com/cilium/cilium/images/builder
+  - ./install-protoc.sh
+  - ./install-protoplugins.sh
+---
+artifact: {{ $.ModuleName }}/{{ $.ImageName }}-gops-artifact
+from: {{ $.Images.BASE_GOLANG_19_BULLSEYE }}
+mount:
+- fromPath: ~/go-pkg-cache
+  to: /go/pkg
+shell:
+  beforeInstall:
+  - apt-get update
+  - apt-get install -y binutils-aarch64-linux-gnu binutils-x86-64-linux-gnu curl
+  - mkdir -p /go/src/github.com/cilium/cilium/images/runtime
+  - curl -sSfL https://raw.githubusercontent.com/cilium/cilium/2a518bb669db2de4677acb952a7eef4f269fcb48/images/runtime/build-gops.sh -o /go/src/github.com/cilium/cilium/images/runtime/build-gops.sh
+  - curl -sSfL https://raw.githubusercontent.com/cilium/cilium/17a78a2325a18793388428402daf063de7d72100/images/runtime/download-cni.sh -o /go/src/github.com/cilium/cilium/images/runtime/download-cni.sh
+  - curl -sSfL https://raw.githubusercontent.com/cilium/cilium/a1046bcc10b75c7130f0579d40d0529c85e1fe56/images/runtime/cni-version.sh -o /go/src/github.com/cilium/cilium/images/runtime/cni-version.sh
+  - chmod +x /go/src/github.com/cilium/cilium/images/runtime/*.sh
```

Comment:
I wish to have these commitSha as configurable variable which always sticks to the release version

Reply:
ok, i'll do it in the next release